### PR TITLE
Feature/floating contact

### PR DESCRIPTION
**Pull request by paulAcato**
The Floating contact template for the HTML and CSS for the WordPress block.

Sadly for the LiveCom part isn't working jet. Therefor we're taking the risk that this must be completely refactored. Yesterday we contacted LiveCom again and kinda received the answer: "You have to implement it different" without mentioning how or providing any documents..

### CODE REVIEW

**Review by robbieatacato — COMMENTED** [review]
The readme is empty.

File: components/FloatingContact/src/FloatingContact.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    this.links = [...this.element.getElementsByClassName(`${id}__item-link`)];
+
+    this.toggle.onclick = () => this.toggleEvents();
+  }
+
+  /**
+   * Toggle event. Fires onclick of the label.
+   */
+  toggleEvents() {
+    const open = this.toggle.getAttribute('aria-expanded') === 'false';
```
Comment (robbieatacato) [inline components/FloatingContact/src/FloatingContact.js:38]: I think you meant: `this.toggle.getAttribute('aria-expanded') === 'true';`

File: components/FloatingContact/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+import './index.scss';
```
Comment (robbieatacato) [inline components/FloatingContact/src/index.tsx:1]: This import is already in the bem file.
The import can be removed.

**Review by robbieatacato — CHANGES_REQUESTED** [review]

File: components/FloatingContact/src/FloatingContact.js (hunk 1 above)
Comment (paulAcato) [reply components/FloatingContact/src/FloatingContact.js:38]: No, i've checked the states again. toggleEvents is fired onclick of the button. We want the current-state before updating it.

Will rename it to `state`

**Review by robbieatacato — APPROVED** [review]

**Pull request merged**